libpyEM/libpyTypeConverter2.cpp

```diff
@@ -44,6 +44,10 @@
 using namespace boost::python;
 namespace np = boost::python::numpy;
 
+int init_numpy()
+{
+    import_array();
+}
```

Comment:
This function expects `return`.

Reply:
Yeah that should not be like this. Thanks for identifying the problem